templates/components/search/controls.html.twig

```diff
@@ -95,12 +95,15 @@
             {% endif %}
         </div>
 
-
         {% if mainform and showaction %}
-        <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}">
-        </form>
+            <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}">
+            </form>
         {% endif %}
 
+        <div class="status-area">
```

Comment:
Maybe something like that to draw attention:

```suggestion
        <div class="status-area alert alert-danger p-1 m-0">
```

![image](https://github.com/glpi-project/glpi/assets/418844/47df8326-e586-41b4-a072-82d23c8a42da)